pkg/tests/archive_test.py

```diff
@@ -227,19 +227,33 @@ def testMergeTarRelocated(self):
       f.add_tar(datafile, name_filter=lambda n: n != "./b", root="/foo")
     self.assertTarFileContent(self.tempfile, content)
 
-  def testPreserveInputTarMtimes(self):
-    input_tar_path = os.path.join(testenv.TESTDATA_PATH, "tar_test.tar")
+  def testDefaultMtimeNotProvided(self):
     with archive.TarFileWriter(self.tempfile) as f:
-      f.add_tar(input_tar_path)
+      self.assertEqual(f.default_mtime, 0)
+
+  def testDefaultMtimeProvided(self):
+    with archive.TarFileWriter(self.tempfile, default_mtime=1234) as f:
+      self.assertEqual(f.default_mtime, 1234)
 
-    with tarfile.open(input_tar_path, "r:") as input_tar:
-      input_mtimes = {}
-      for input_file in input_tar:
-        input_mtimes[input_file.name] = input_file.mtime
+  def testPortableMtime(self):
+    with archive.TarFileWriter(self.tempfile, default_mtime="portable") as f:
+      self.assertEqual(f.default_mtime, 946684800)
 
-    with tarfile.open(self.tempfile, "r:") as output_tar:
-      for output_file in output_tar:
-        self.assertEqual(input_mtimes[output_file.name], output_file.mtime)
+  def testPreserveTarMtimesTrueByDefault(self):
+    with archive.TarFileWriter(self.tempfile) as f:
+      input_tar_path = os.path.join(testenv.TESTDATA_PATH, "tar_test.tar")
```

Comment:
Where did testenv come from?  The tests are not passing because of this not being defined.  Suggest
      input_tar_path = self.data_files.Rlocation(
          os.path.join("rules_pkg", "tests", "testdata", "tar_test.tar"))

Reply:
Done; I was running the tests in another environment that had that environment setup. Is there a good way for me to see the test run results for these repo?

Reply:
I sync to head, patch in the changes and do `bazel test ...`
We should get that running on the Bazel CI systems. There is a bug for that. #138